### ISSUE TITLE
Build occurs twice during deployment with cloudflare workers

### ISSUE THREAD

**Issue by aiji42** [remix-run/remix#2238]
### What version of Remix are you using?

1.2.3

### Steps to Reproduce

1. Select Cloudflare Workers for runtime.
2. Run `yarn deploy` to deploy.

### Expected Behavior

The build is executed only once.

### Actual Behavior

The build is executed twice.

The deploy script in package.json generated from the template is as follows, and because `wrangler.toml` also contains commands regarding build, it is executed twice.
`"deploy": "npm run build && wrangler publish"`

The start command likewise performs the build again at server start, even though there are pre-built artifacts.

**Comment by aiji42** [comment]
The build description should be removed from `wrangler.toml`.

```toml

name = "remix-cloudflare-workers"
type = "javascript"

zone_id = ""
account_id = ""
route = ""
workers_dev = true

[site]
bucket = "./public"
entry-point = "."

[build]                                         <-
command = "npm run build"    <-

[build.upload]
format="service-worker"
```

**Comment by aiji42** [comment]
There are several problems caused by the double execution of builds.
- Longer CI run times
- If you use environment variables in `remix.config.js` and CI has separate build and deploy steps, you must use the same environment variables in both steps.

**Comment by kiliman** [comment]
I also noticed that when running the GitHub action from `cloudflare/wrangler-action@1.3.0`, this fails because the action uses `node v12` which is not recent enough for `remix build`.

**Comment by machour** [comment]
Duplicate of #2078
